cms/djangoapps/contentstore/utils.py

```diff
@@ -455,3 +457,10 @@ def get_visibility_partition_info(xblock):
         "has_selected_groups": has_selected_groups,
         "selected_verified_partition_id": selected_verified_partition_id,
     }
+
+
+def is_self_paced(course):
+    """
+    Returns True if course is self-paced, False otherwise.
+    """
+    return course.self_paced and SelfPacedConfiguration.current().enabled if course else False
```

Comment:
nit: I think this is cleaner:
```return course and course.self_paced and SelfPacedConfiguration.current().enabled```

BTW, what happens if there is no entry in `SelfPacedConfiguration`?